docs/user/dql/functions.rst

```diff
@@ -1832,3 +1832,82 @@ Description
 Specifications:
 
 1. ISNULL(ES_TYPE) -> INTEGER
+
+
+CASE
+----
+
+Description
+>>>>>>>>>>>
+
+``CASE`` statement has two forms with slightly different syntax: Simple Case and Searched Case.
+
+Simple case syntax compares a case value with each compare value in ``WHEN`` clause and return its result if matched. Otherwise, result in ``ELSE`` clause is returned (or ``NULL`` if absent). Note that case value, compare value and result can be any expression::
+
+   CASE case_value
```

Comment:
The case_value and compare_value could be expression, right? Could we refer the T-SQL definition https://docs.microsoft.com/en-us/sql/t-sql/language-elements/case-transact-sql?view=sql-server-ver15

Reply:
sure, will do. thanks!